src/common/stats/StatsReporter.cpp

```diff
@@ -0,0 +1,63 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include "http/HttpClient.h"
+#include "network/NetworkUtils.h"
+#include "stats/StatsReporter.h"
+#include <cinttypes>
+#include <folly/dynamic.h>
+
+DEFINE_string(statistics_reporter_address, "",
+        "URL address that reporting statistics to."
+		"In OpenFalcon case, this may be http://127.0.0.1:1988/v1/push");
+
+namespace nebula {
+namespace stats {
+
+void reportMetrics(const MetricRecored& record) {
+    if (!FLAGS_statistics_reporter_address.empty()) {
+        std::string json;
+        generateJson(record, &json);
+        StatusOr<std::string> result = http::HttpClient::postData(
```

Comment:
Very good feature.

But i have concern about the push mode maybe cause lots of subprocesses (That's what we did inside HttpClient) on server side. 

Could we replace the push mode with pull?

Reply:
make sense, let me try

Reply:
Add StorageHttpStatsHandler to exposing rocksdb statistics through webservice, so we can pull metrics from it.
Do you think there is a need to reserve the push logic(since it's configurable, and disabled by default), or just discard it. @dangleptr

Reply:
IMO we should remove the useless code.  There are too many codes already....

Reply:
OK, remove the push logic to make the PR simple.